src/main/java/no/nav/finnkandidatapi/midlertidigutilgjengelig/MidlertidigUtilgjengelig.java

```diff
@@ -26,4 +30,21 @@ public static MidlertidigUtilgjengelig opprettMidlertidigUtilgjengelig(Midlertid
                 .registrertAvNavn(veileder.getNavn())
                 .build();
     }
+
+    public static Optional<String> finnMidlertidigUtilgjengeligFilter(Optional<MidlertidigUtilgjengelig> midlertidigUtilgjengelig) {
+        if (midlertidigUtilgjengelig.isEmpty()) {
+            return Optional.empty();
+        }
+        LocalDateTime tilDato = midlertidigUtilgjengelig.get().getTilDato();
+        LocalDateTime nå = LocalDateTime.now();
+        if (tilDato == null) {
+            return Optional.empty();
+        } else if (tilDato.isBefore(nå)) {
+            return Optional.empty();
+        } else if (tilDato.isBefore(nå.plusWeeks(1))) {
```

Comment:
Er denne for nøyaktig, burde jeg her heller ta bort timer og minutt?

Reply:
Jeg har ikke helt oversikt over koden enda, så mulig jeg missforstår her.
Men vi vil vel ha at en kandidat ha denne statusen fra kl 00:00/når jobben kjøres på natten?
Da har du vel rett i at vi ikke kan ha med timer og minutt...